service/src/main/resources/application.yml

```diff
@@ -9,3 +9,14 @@ management:
       exposure:
         include: info, health, metrics, mappings
 
+mongodb:
+  uri: ${GP2GP_MONGO_URI:mongodb://localhost:27017}
+  database: ${GP2GP_MONGO_DATABASE_NAME:gp2gp}
+  host: ${GP2GP_MONGO_HOST:}
+  port: ${GP2GP_MONGO_PORT:}
+  username: ${GP2GP_MONGO_USERNAME:}
+  password: ${GP2GP_MONGO_PASSWORD:}
+  options: ${GP2GP_MONGO_OPTIONS:}
+  autoIndexCreation: ${GP2GP_MONGO_AUTO_INDEX_CREATION:true}
+  ttl: ${GP2GP_MONGO_TTL:P30D}
+  cosmosDbEnabled: ${GP2GP_COSMOS_DB_ENABLED:false}
```

Comment:
Please add environment variables to ./README.md +  update ./docker/docker-compose.yml with these new variables